packages/layout/src/helpers/breakpoints.ts

```diff
@@ -0,0 +1,52 @@
+export type ResponsiveObject<T> = { sm?: T; md?: T; lg?: T; xl?: T; xxl?: T };
```

Comment:
Looking at some of these helper functions made me wonder if there's any merit to writing some simple tests to verify / strap them in? Not suggesting we must btw, more of a thought that occurred to me as i'm looking over, and whether they would be helpful to us and for the future - especially since i don't necessarily grok everything they are doing from a glance.

Reply:
Yeah I was going to write tests for all of these once we get functionality finalized!